verification/VerifyImputationBeagle.wdl

```diff
@@ -0,0 +1,78 @@
+version 1.0
+
+import "../verification/VerifyTasks.wdl" as Tasks
+
+## Copyright Broad Institute, 2018
+##
+## This WDL script is designed to verify (compare) the outputs of an ArrayWf wdl.
+##
+##
+## Runtime parameters are optimized for Broad's Google Cloud Platform implementation.
+## For program versions, see docker containers.
+##
+## LICENSING :
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+workflow VerifyImputationBeagle {
+  input {
+    Array[File] truth_metrics
+    Array[File] test_metrics
+
+    File truth_vcf
+    File test_vcf
+    File test_vcf_index
+    File truth_vcf_index
+
+    Boolean? done
+  }
+
+  String bcftools_docker_tag = "us.gcr.io/broad-gotc-prod/imputation-bcf-vcf:1.0.7-1.10.2-0.1.16-1669908889"
+
+  scatter (idx in range(length(truth_metrics))) {
+    call CompareImputationMetrics {
+      input:
+        test_metrics = test_metrics[idx],
+        truth_metrics = truth_metrics[idx]
+    }
+  }
+
+  call Tasks.CompareVcfsAllowingQualityDifferences as CompareOutputVcfs {
```

Comment:
do we really want to allow quality differences?  i woudl expect the runs to be exactly the same.  def worth a chat if you're seeing only qual differences

Reply:
the odd thing is that this task will still fail if there's quality differences; the log will just include the line "but the only difference was in QUAL". functionally the only way you get a successful exit code in the task is if the entire thing is the same. 

maybe that's a bug we want to fix? or just use the CompareVcfs task here in warp?

Reply:
oh...yea thats def a bug.  I would probably just use the CompareVcfs task in warp.  Up to you if you wanna fix the bug but im assuming someone who was actually relying on it would fix it themselves....